pages/linux/pmount.md

```diff
@@ -0,0 +1,15 @@
+# pmount
+
+> Mount arbitrary hotpluggable devices as normal user.
+
+- Mount a device below /media/ (using device as mount point):
+
+`pmount {{path/to/device}}`
```

Comment:
the device is a device unit right? so:
```suggestion
`pmount {{/dev/sdX}}`
```

Reply:
I'm not sure about this one, since SD cards (which I often pmount) are at `/dev/mmcblk...`.

Reply:
I'm unsure about this one, since for example SD cards are named differently.

Reply:
Well, we can use `path/to/file_or_directory` when you can use both as an argument. So, I think we can use:
```suggestion
`pmount {{/dev/sdX_or_mmcblkX}}`
```

Reply:
`_or_whatever`? I agree that `path/to/device` is bad, but maybe you're too specific here. Is it possible to use a general form like `/dev/device` or `/dev/X`?

Reply:
:thinking: `dev` is `device` so `/dev/device` is redundant. `/dev/mmcblk` is a helper to guide the reader to find the SD card reader. Maybe we can use:
```suggestion
`pmount {{/dev/to/block/device}}`
```
And it reads similar to `path to file`

Reply:
This looks fine to me.